mandatory/1-fix-functions.js

```diff
@@ -10,19 +10,18 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(mood) {
```

Comment:
this is not good naming, `mood` is the name of both the function and the argument, this is confusing